DependencyInjection/DoctrineExtension.php

```diff
@@ -942,8 +942,9 @@ private function registerMetadataPhpArrayCaching(string $entityManagerName, Cont
         $decoratedMetadataCacheServiceId = (string) $container->getAlias($metadataCacheAlias);
         $phpArrayCacheDecoratorServiceId = $decoratedMetadataCacheServiceId . '.php_array';
         $phpArrayFile                    = '%kernel.cache_dir%' . sprintf('/doctrine/orm/%s_metadata.php', $entityManagerName);
+        $cacheWarmerServiceId            = sprintf('doctrine.orm.metadata_cache_warmer.%s', $entityManagerName);
```

Comment:
I might be nitpicking, but can we please use similar approach here as on line 941? Meaning, this would be 
```suggestion
        $cacheWarmerServiceId            = $this->getObjectManagerElementName($entityManagerName . '_metadata_cache_warmer');
```

This would follow same naming convention